### PR TITLE
[FIZZ-202] Canvas Packaging

### PR DESCRIPTION
**Pull request by stephenfuqua**
Includes some updates in the `extractor-lib` and `google-classroom-extractor` based on lessons learned with publishing.

Also, somehow we ended up with both a `settings.kts` and `Settings.kts` file. The lowercase is correct, and I deleted the uppercase version.

### CODE REVIEW

File: src/schoology-extractor/pyproject.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@ python-dotenv = "^0.14.0"
 opnieuw = "^1.1.0"
 SQLAlchemy = "^1.3.20"
 errorhandler = "^2.0.1"
-edfi-lms-extractor-lib = {version = "1.0.0a1", allow-prereleases = true}
+edfi-lms-extractor-lib = {version = "^1.0.0a0", allow-prereleases = true}
```
[the remark this answers is not in the document]
Comment (stephenfuqua) [reply src/schoology-extractor/pyproject.toml:19]: I rather like the notion of saying "get me the latest prerelease" but I see and agree with your point.
```suggestion
edfi-lms-extractor-lib = {version = "1.0.0a2", allow-prereleases = true}
```

File: src/canvas-extractor/pyproject.toml [hunk 2]
```diff
@@ -20,7 +20,7 @@ SQLAlchemy = "^1.3.20"
 opnieuw = "^1.1.0"
 errorhandler = "^2.0.1"
 ConfigArgParse = "^1.2.3"
-edfi-lms-extractor-lib = {version = "1.0.0a1", allow-prereleases = true}
+edfi-lms-extractor-lib = {version = "^1.0.0a0", allow-prereleases = true}
```
Comment (stephenfuqua) [inline src/canvas-extractor/pyproject.toml:23]:
```suggestion
edfi-lms-extractor-lib = {version = "1.0.0a2", allow-prereleases = true}

```

File: src/google-classroom-extractor/pyproject.toml [hunk 3]
```diff
@@ -22,7 +23,7 @@ xxhash = "^2.0.0"
 numpy = "1.19.3"
 errorhandler = "^2.0.1"
 ConfigArgParse = "^1.2.3"
-edfi-lms-extractor-lib = {version = "1.0.0a1", allow-prereleases = true}
+edfi-lms-extractor-lib = {version = "^1.0.0a0", allow-prereleases = true}
```
Comment (stephenfuqua) [inline src/google-classroom-extractor/pyproject.toml:26]:
```suggestion
edfi-lms-extractor-lib = {version = "1.0.0a2", allow-prereleases = true}
```

**Review by bradbanister — APPROVED** [review]
Looks good

**Pull request merged**